### PR TITLE
draw all available shortcuts in the right corner

### PR DESCRIPTION
**Pull request by spjoe**
Have done this [Todo](https://github.com/SFTtech/openage/compare/master...spjoe:feature/shortcutRendering?expand=1#diff-3cd26de8330566a41cadb63f7de53cccL146)

Features:
+ gets binded actions directly from global input context and current output mode.
+ gets shortcut configuration from input manager.

EditorMode
CreationMode
ActionMode

### CODE REVIEW

File: libopenage/input/action.cpp [hunk 1]
```diff
@@ -9,7 +9,11 @@ namespace input {
 
 
 int action_hash::operator()(const action_t& a) const {
-	return std::hash<int>()(static_cast<int>(a));
+	return std::hash<int>()(a.key);
+}
+
+bool action_t::operator==(const action_t& other) const {
+  return key == other.key;
```
Comment (TheJJ) [inline libopenage/input/action.cpp]: whee indentation :)

**Comment by TheJJ** [comment]
Very nice, is probably very helpful for endusers.

Please keep our codestyle guideline in mind :stuck_out_tongue_winking_eye: 
https://github.com/SFTtech/openage/tree/master/doc/code_style

Namely, after `for`, `if`, `while`, there should be a space after that word, so `for (;;) {` and `if (cond) {` are correct, and `if(cond){` is missing two spaces.
It's of course not wrong what you did, but we wanna keep the style consistent, right? :sunglasses:

**Comment by franciscod** [comment]
this is great!

**Comment by TheJJ** [comment]
Thanks man, good job :)

**Pull request closed without merging**